valheim_plus.cfg

```diff
@@ -428,7 +428,7 @@ dontProcessFineWood=false
 dontProcessRoundLog=false
 
 ; The time it takes for the Kiln to produce a single piece of coal in seconds.
-productionSpeed=30
+productionSpeed=16
```

Comment:
Should be 15 as per the default game settings

Reply:
Thanks for checking, you're right!